examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,62 @@
+"""
+===================================
```

Comment:
```suggestion
==========================
```

Reply:
I am unsure if this is the right length. I can't recall if Github renders a monospaced font correctly.

Reply:
I will google and go through it!!